javascript/log.js

```diff
@@ -40,10 +40,42 @@ const success = (event, halted) => {
     .split('-')
     .slice(1)
     .join('_')
+  const output = { reflexId, morph, payload }
+  if (operation !== 'dispatch_event') output.operation = operation
   console.log(
     `\u2193 reflex \u2193 ${target} \u2192 ${selector ||
       '\u221E'}${progress} ${duration}`,
-    { reflexId, morph, operation, halted, payload }
+    output
+  )
+}
+
+const halted = event => {
```

Comment:
I am going to push back for the moment, partially because I know that the `duration` ternary will come out when we can finally remove the isolation mode code, partially out of pragmatism that this is a fussy module that I have primarily maintained, and right now it both works well and makes sense to me.

This is, IMO, an example of functionality where each case is just slightly different enough that trying to make things slightly (and it would be very slight) less repetitive would drastically ramp up the complexity of the solution; right now it's procedural and doesn't require following a call stack.

Reply:
I (partially) agree. Not every duplication is worth resolving, _but_ we could still think of making the logger more OOP, i.e. working with delegates or what have you...

Reply:
Resolving this for now, as refactoring the logger is wildly out of scope for this PR.